### PR TITLE
05_sumAll: Update README.md

### LINKED ISSUE #255: Bug - : 05_sumALL: Returning ERROR with negative numbers
Complete the following REQUIRED checkboxes:
-   [ o] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [ o] The title of this issue follows the `Bug - location of bug: brief description of bug` format, e.g. `Bug - Exercises: File type incorrect for all test files`

The following checkbox is OPTIONAL:

-   [ ] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Bug:**
The 4th test for this task checks that the function returns ERROR with negative numbers. However, the question did not state that the integers had to be non-negative, and it is in fact possible to obtain a proper answer even with negative numbers.

**2. How To Reproduce:**
1. Code without error checking for negative numbers.
2. 4th test fails

**3. Expected Behavior:**
1. Code without error checking for negative numbers.
2. 4th test should pass, with an answer of -45

**4. Desktop/Device:**

-   Device: Virtual Machine
-   OS: Ubuntu
-   Browser: NIL
-   Version: NIL

**5. Additional Information:**

### PR DESCRIPTION
**Pull request by shubhatRashid**
## Because

The test cases for the sumAll question suggested that the integers needed to be positive however no such thing was mentioned in question statement .

## This PR

Changes the question statement which previously stated to 'form a function for integers' to 'form a function for positive integers'

## Issue

Closes #255 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `01_helloWorld: Update test cases`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If this PR includes any changes that affect the solution of an exercise, I've also updated the solution in the `/solutions` folder

### CODE REVIEW

File: 04_removeFromArray/removeFromArray.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-const removeFromArray = function() {
-
+const removeFromArray = function(...arg) {
+
```
Comment (MaoShizhong) [inline 04_removeFromArray/removeFromArray.js:2]: Reverting unrelated file to original state
```suggestion
const removeFromArray = function() {

```

File: 04_removeFromArray/removeFromArray.spec.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ describe('removeFromArray', () => {
   test.skip('ignores non present values', () => {
     expect(removeFromArray([1, 2, 3, 4], 7, "tacos")).toEqual([1, 2, 3, 4]);
   });
-  test.skip('ignores non present values, but still works', () => {
-    expect(removeFromArray([1, 2, 3, 4], 7, 2)).toEqual([1, 3, 4]);
+  test('ignores non present values, but still works', () => {
+    expect(removeFromArray([1,2, 2, 3, 4], 7, 2)).toEqual([1, 3, 4]);
```
Comment (MaoShizhong) [inline 04_removeFromArray/removeFromArray.spec.js:17]: Reverting unrelated file to original state
```suggestion
  test.skip('ignores non present values, but still works', () => {
    expect(removeFromArray([1, 2, 3, 4], 7, 2)).toEqual([1, 3, 4]);
```

File: 05_sumAll/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Exercise 05 - sumAll
 
-Implement a function that takes 2 integers and returns the sum of every number between(and including) them:
+Implement a function that takes 2 positive integers and returns the sum of every number between(and including) them:
```
Comment (MaoShizhong) [inline 05_sumAll/README.md:3]: Quick fix
```suggestion
Implement a function that takes 2 positive integers and returns the sum of every number between (and including) them:
```

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Thanks @shubhatRashid. In the future, could you please comment on the relevant issue and request to be assigned? Then only open a PR after you've been assigned by a maintainer as per our [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md#being-assigned-an-issue).

Just need a quick spacing fix as well as removing the two unrelated changes that accidentally made their way into this commit.

**Comment by shubhatRashid** [comment]
Hey @MaoShizhong ,
I will keep the guidelines under consideration next time, i have made the requested changes .

File: 04_removeFromArray/removeFromArray.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 const removeFromArray = function() {
-
+
```
Comment (MaoShizhong) [inline 04_removeFromArray/removeFromArray.js:2]:
```suggestion

```

File: 04_removeFromArray/removeFromArray.spec.js [hunk 5]
```diff
@@ -14,7 +14,7 @@ describe('removeFromArray', () => {
     expect(removeFromArray([1, 2, 3, 4], 7, "tacos")).toEqual([1, 2, 3, 4]);
   });
   test.skip('ignores non present values, but still works', () => {
-    expect(removeFromArray([1, 2, 3, 4], 7, 2)).toEqual([1, 3, 4]);
+    expect(removeFromArray([1,2, 3, 4], 7, 2)).toEqual([1, 3, 4]);
```
Comment (MaoShizhong) [inline 04_removeFromArray/removeFromArray.spec.js:17]:
```suggestion
    expect(removeFromArray([1, 2, 3, 4], 7, 2)).toEqual([1, 3, 4]);
```

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
The following bits are still necessary to remove the unrelated files from the PR

**Comment by shubhatRashid** [comment]
Hey @MaoShizhong ,
Sorry for my silly mistakes,
i have made the necessary changes.

**Review by MaoShizhong — APPROVED** [review]
Many thanks!

**Comment by shubhatRashid** [comment]
My pleasure...

**Pull request merged**